woocommerce/woocommerce
# Adjust input field height and input label text size

### Changes proposed in this Pull Request:





This PR introduces the following changes:

- Ensure that text input fields have a consistent height of 50px.
- Update the labels of text fields to use a font size of approximately 12px when the input field contains text (with a scaling factor of 0.75 when text is present).

Closes #49477.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the frontend.
2. Add a product to the cart.
3. Go to the cart page that contains the Cart block.
4. Verify that the coupon input field has a height of 50px.
5. Go to the checkout page that contains the Checkout block.
6. Verify that all text input fields and the native select fields have a height of 50px.
7. Verify that the text size of the input field labels is approximately 12px when the input field has a value.

<table>
<tr>
<td valign="top">Before:
<br><br>
<img width="700" alt="Screenshot 2024-07-17 at 12 41 16" src="https://github.com/user-attachments/assets/58aaba3f-ea4c-4010-b9a3-b29e4ab927fc">

</td>
<td valign="top">After:
<br><br>
<img width="700" alt="Screenshot 2024-07-17 at 12 36 14" src="https://github.com/user-attachments/assets/5c2062ae-c54d-48d9-a025-7e6c450169b9">

</td>
</tr>
</table>



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Adjust input field height and input label text size.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
